opendbc/dbc/hyundai_canfd.dbc

```diff
@@ -160,6 +160,7 @@ BO_ 298 LFA: 16 ADRV
  SG_ LKA_ASSIST : 62|1@1+ (1,0) [0|1] "" XXX
  SG_ LKA_MODE : 24|3@1+ (1,0) [0|7] "" XXX
  SG_ NEW_SIGNAL_2 : 70|2@0+ (1,0) [0|3] "" XXX
+ SG_ NEW_SIGNAL_4 : 72|4@1+ (1,0) [0|15] "" XXX
```

Comment:
Can we give these semi-relevant, self-describing names?

Reply:
The added [`NEW_SIGNAL_4`](https://github.com/commaai/opendbc/pull/1575/files#diff-256b6a0bc4c776172e296de10551d95f65f5013dbbf8364478d35f07fffc0828R47) & [`NEW_SIGNAL_5`](https://github.com/commaai/opendbc/pull/1575/files#diff-256b6a0bc4c776172e296de10551d95f65f5013dbbf8364478d35f07fffc0828R174) are like the other `NEW_SIGNAL_`'s defined, just signals we set to match stock. Don't have anything other than that at the moment.

Reply:
I know, but they should ideally be as descriptive as possible, or comments added in the DBC to hint at what we suspect t hey might do.

Reply:
Added comments, values seem to always be the same, graph shows car on, off, then on again. We can remove if this isn't good enough. <img width="1052" alt="Screenshot 2025-01-27 at 23 10 45" src="https://github.com/user-attachments/assets/c5afc3cd-e498-41f5-9557-818597d78ecb" />